integration/token/fungible/views/issue.go

```diff
@@ -3,29 +3,28 @@ Copyright IBM Corp. All Rights Reserved.
 
 SPDX-License-Identifier: Apache-2.0
 */
+
 package views
 
 import (
 	"encoding/json"
 	"fmt"
 
-	"github.com/hyperledger-labs/fabric-token-sdk/token"
-
 	view2 "github.com/hyperledger-labs/fabric-smart-client/platform/view"
-	"github.com/hyperledger-labs/fabric-token-sdk/token/services/network"
-
-	token2 "github.com/hyperledger-labs/fabric-token-sdk/token/token"
-
 	"github.com/hyperledger-labs/fabric-smart-client/platform/view/services/assert"
 	"github.com/hyperledger-labs/fabric-smart-client/platform/view/view"
-
+	"github.com/hyperledger-labs/fabric-token-sdk/token"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/network"
 	"github.com/hyperledger-labs/fabric-token-sdk/token/services/ttx"
+	token2 "github.com/hyperledger-labs/fabric-token-sdk/token/token"
 )
 
 // IssueCash contains the input information to issue a token
 type IssueCash struct {
 	// Anonymous set to true if the transaction is anonymous, false otherwise
 	Anonymous bool
+	// Auditor is the name of the auditor identity
```

Comment:
is it name or enrolment ID?

Reply:
it is just the name/label in the configuration used to identify the FSC auditor identity.